wandb/sdk/launch/docker.py

```diff
@@ -68,17 +64,32 @@ def generate_docker_image(project: _project_spec.Project, entry_cmd):
     return image_id[0]
 
 
-def build_docker_image(project: _project_spec.Project, repository_uri, base_image, api):
+def build_docker_image(project: _project_spec.Project, docker_repository_uri, base_image, api):
```

Comment:
oh i changed all these varnames in the docker pr as well to just `name`, it's not really a uri

Reply:
I can also change this to name to avoid conflict